common/scala/src/main/scala/whisk/core/entity/ExecManifest.scala

```diff
@@ -41,11 +43,11 @@ protected[core] object ExecManifest {
    * singleton Runtime instance.
    *
    * @param config a valid configuration
-   * @param localDockerImagePrefix optional local docker prefix, permitting images matching prefix to bypass docker pull
    * @return the manifest if initialized successfully, or an failure
    */
-  protected[core] def initialize(config: WhiskConfig, localDockerImagePrefix: Option[String] = None): Try[Runtimes] = {
-    val mf = Try(config.runtimesManifest.parseJson.asJsObject).flatMap(runtimes(_, localDockerImagePrefix))
+  protected[core] def initialize(config: WhiskConfig): Try[Runtimes] = {
+    val rmc = loadConfigOrThrow[RuntimeManifestConfig](ConfigKeys.runtimes)
+    val mf = Try(config.runtimesManifest.parseJson.asJsObject).flatMap(runtimes(_, rmc))
```

Comment:
if the manifest itself is moved to pureconfig, then can nuke whisk config here entirely - is that feasible?

Reply:
I think moving the manifest itself to pure config isn't a good idea.  We override it in our deployments and overriding a complex structure via an environment variable is going to be annoying.  Better to keep the manifest itself out of the docker images.

Reply:
i was thinking more about making the value a path rather than the structure itself - but don't know if that's feasible and if the file path resolution would be an issue.